modules/auxiliary/gather/browser_sop_bypass.rb

```diff
@@ -0,0 +1,53 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'           => "Browser SOP Bypass",
+        'Description'    => %q(
+          This module do a server-redirect combined with a data-URI end up bypassing the Same Origin Policy,which leads to all kind of vulnerabilities like stealing user passwords.
+        ),
+        'License'        => MSF_LICENSE,
+        'Author'         => [
+          'Dhiraj Mishra',
+        ],
+        'References'     => [
+        ['URL', 'http://fr.0day.today/exploit/description/28434'],
```

Comment:
As stated on #8977, the reference doesn't actually describe the vulnerability, and further, it's super unclear what the reuse conditions are imposed or expected by the site maintainers.

Reply:
Also, can you point to the English version instead? Metasploit tends to prefer that.